grails-app/controllers/au/org/ala/specieslist/SpeciesListController.groovy

```diff
@@ -329,13 +329,13 @@ class SpeciesListController {
 
     def getGuidsForList(id, limit){
         def fqs = params.fq?[params.fq].flatten().findAll{ it != null }:null
-        def baseQueryAndParams = queryService.constructWithFacets(" from SpeciesListItem sli ",fqs, params.id)
+        def baseQueryAndParams = queryService.constructWithFacets(" from SpeciesListItem sli ", fqs, params.id, null)
```

Comment:
I'm pretty sure the previous code without last argument is valid in groovy. Was there a reason you made this change? Did it throw an error?

Reply:
constructWithFacets signature changed, added "q" which is the last parameter.

Reply:
See my comments under the service method - will still match the method without the last arg - groovy does a best match thing, unlike Java.